src/components/MemberSearchPage/FilterTagContainer.js

```diff
@@ -4,21 +4,32 @@ import FilterTagContent from 'components/MemberSearchPage/FilterTagContent';
 
 const FilterTagContainer = ({ filters, clearTag }) => {
   return (
-    <Row className={'flex'} gutter={16} style={{ paddingTop:32, marginLeft: 0, marginRight: 0 }}>
+    <Row className={'flex'} gutter={16} style={{ paddingTop: 32, marginLeft: 0, marginRight: 0 }}>
```

Comment:
since you already use css why not extracting the inline-style into a css class?

Reply:
Its not really possible in this case...